content/about/_index.md

```diff
@@ -1,20 +1,33 @@
-4. Perform an **A/B/N test** with multiple versions -- a baseline and multiple candidates. Iter8 will identify and shift application traffic safely and gradually to the winner.
+<!-- Software engineers use a variety of continuous experimentation strategies to achieve desirable outcomes for their businesses. The desired outcome could be safely releasing a new version in production while ensuring it meets various service level objectives (SLOs), or finding the `best` version among multiple competing versions of a microservice using live application traffic, or performance testing a version in a dev/test/staging environment before releasing in production.  -->
 
-Under the hood, iter8 uses advanced Bayesian learning techniques coupled with multi-armed bandit approaches to compute a variety of statistical assessments for your microservice versions, and uses them to make robust traffic control and rollout decisions.
+Iter8 is an open source toolkit for continuous experimentation on kubernetes, which makes it easy for you to deliver high-impact code changes within your microservices applications in an agile manner while eliminating the risk. Using iter8 machine learning (ML)-driven experimentation capabilities, you can safely and rapidly orchestrate various types of live experiments, gain key insights into the behavior of your microservices, and rollout the best versions of your microservices in an automated, principled, and statistically robust manner.
 
-![iter8pic]({{< resourceAbsUrl path="images/iter8pic.png" >}})
+## What is an iter8 experiment?
+Use an `iter8 experiment` to safely expose competing versions of a service to application traffic, gather in-depth assessments about key performance and business metrics for your microservice versions, and intelligently rollout the best version of your service. Iter8's expressive model of experimentation supports a diverse variety of experiments. The four main kinds of experiments in iter8 are as follows.
+
+1. Run a **performance test** with a single version of a microservice. `iter8` will verify if the version meets the criteria you specify in the experiment.
+2. Perform a **canary release** with two versions, a baseline and a candidate. Iter8 will shift application traffic in a safe and progressive manner to the candidate, if the candidate meets the criteria you specify in the experiment.
+3. Perform an **A/B rollout** with two versions -- a baseline and a candidate. Iter8 will identify and shift application traffic safely and gradually to the `winner`, where the winning version is defined by the criteria you specify in the experiment.
+4. Perform an **A/B/n rollout** with multiple versions -- a baseline and multiple candidates. Iter8 will identify and shift application traffic safely and gradually to the winner.
```

Comment:
winner here should be code to match item 3.  Or both not...

Reply:
Fixed both.